src/definition/users/IBotUser.ts

```diff
@@ -0,0 +1,6 @@
+import { UserType } from './UserType';
+import { IUser } from './IUser';
+
+export interface IBotUser extends Omit<IUser, 'emails' | 'utcOffset'> {
```

Comment:
Why omitting the `utcOffset`? If it is a federated user, such as teams or slack, then the app could in theory provide that value.

Reply:
Make sense!